[WIP] Certificate templates

ateucher opened the pull request.

ateucher commented on R/create_certificate.R:
```diff
@@ -1,67 +1,126 @@
+    ...
+  )
+
+  if (output_dir != ".") {
```
Good catch!

ateucher commented on R/create_certificate.R:
```diff
@@ -72,98 +131,161 @@ create_certificate <- function(cohort_name,
+          cohort_type = cohort_type,
+          output_dir = output_dir
+        )
+      },
```
Good idea, thanks!

ateucher commented:
@seankross thanks for the review! I have addressed all of your comments I think; feel free to do another round if you would like!

ateucher commented:
Thanks @seankross!

seankross commented on R/create_certificate.R:
```diff
@@ -72,98 +129,189 @@ create_certificate <- function(cohort_name,
 #' \dontrun{
```
In this example you should add `googlesheets4::gs4_auth()` before you call `read_sheet()`.

seankross commented on R/create_certificate.R:
```diff
@@ -72,98 +129,189 @@ create_certificate <- function(cohort_name,
 #' \dontrun{
 #' registry <- read_sheet("path-to-registry")
```
```suggestion
#' registry <- googlesheets4::read_sheet("path-to-registry")
```

seankross commented on R/create_certificate.R:
```diff
@@ -72,98 +129,189 @@ create_certificate <- function(cohort_name,
 #' \dontrun{
 #' registry <- read_sheet("path-to-registry")
 #' participants <- read_sheet("path-to-participants")
```
```suggestion
#' participants <- googlesheets4::read_sheet("path-to-participants")
```

seankross reviewed (changes requested):
Just a few suggestions, otherwise looks good.

ateucher commented:
Thanks @seankross! I've addressed your comments, all good. I also had to make a change to the call to `quarto::quarto_render()` due to a recent breaking change in the quarto R package (https://github.com/quarto-dev/quarto-r/pull/257).

LMK if you think it's good to merge now!
